### PR TITLE
Add folder support to dicom2img (display all folder images in collage)

### PR DESCRIPTION
**Pull request by timcogan**

### CODE REVIEW

File: dicom_utils/cli/dicom2img.py [hunk 1]
```diff
@@ -26,10 +28,50 @@ def get_parser(parser: ArgumentParser = ArgumentParser()) -> ArgumentParser:
     return parser
 
 
+def to_collage(images: List[ndarray]) -> ndarray:
+    num_images = len(images)
+
+    assert num_images != 0, "There must be at least one image."
+    assert all(len(i.shape) == 3 for i in images), "The images must have 3 channels."
```
Comment (TidalPaladin) [inline dicom_utils/cli/dicom2img.py:35]: Should this assertion message be reworded as "3 dimensions"? 3 channels implies RGB

File: dicom_utils/cli/dicom2img.py [hunk 2]
```diff
@@ -80,19 +118,25 @@ def dicom_to_image(
...
-        raise FileNotFoundError(source)
-
     # handle case where output path is a dir
     if dest is not None and dest.is_dir():
-        dest = Path(dest, source.stem).with_suffix(".png")
+        dest = Path(dest, path.stem).with_suffix(".png")
 
-    with pydicom.dcmread(source) as dcm:
-        dicom_to_image(dcm, dest, args.split, args.fps, args.quality, not args.noblock)
+    dcms = [pydicom.dcmread(source) for source in path_to_sources(path)]
```
Comment (TidalPaladin) [inline dicom_utils/cli/dicom2img.py:138]: Should a try/except block be added to `dcms` list building? If we want to visualize a directory but that directory contains one badly-formatted / unreadable dicom, we won't be able to visualize anything. Seems preferable to log such errors and continue on, visualizing whatever we were able to read.

File: dicom_utils/cli/dicom2img.py (hunk 1 above)
Comment (timcogan) [reply dicom_utils/cli/dicom2img.py:35]: Yes, good catch

File: dicom_utils/cli/dicom2img.py (hunk 2 above)
Comment (timcogan) [reply dicom_utils/cli/dicom2img.py:138]: Yes, I think that makes sense. I'll add that in.

**Pull request merged**